integration-test/helpers/pages.js

```diff
@@ -252,7 +281,8 @@ export function loginPage (page, server, opts = {}) {
          */
         async assertParentOpened () {
             const credsCalls = await mockedCalls(page, ['getSelectedCredentials'], true)
-            expect(credsCalls.length).toBe(5)
+            const hasSucceeded = credsCalls.some((call) => call[2]?.some(({type}) => type === 'ok'))
```

Comment:
NAB: I regret representing the mocks as a tuple -> that `[2]` there could of been named instead. Let's follow up at some point to remove the tuples and do it with object literals